etc/gen_atlas_search/src/main.rs

```diff
@@ -77,17 +91,17 @@ impl Operator {
             #[allow(missing_docs)]
             pub struct #name_ident;
```

Comment:
Suggest adding a basic doc for these structs, something like:
```
/// {operator} Atlas Search operator. Construct with {link to constructor function}.
```

This is primarily motivated by the generated docs, which look a little bare in their current state:
<img width="1008" height="852" alt="Screenshot 2025-09-05 at 15 04 50" src="https://github.com/user-attachments/assets/6ea281a1-8862-4a87-b164-c5ac8b4bc683" />